.github/pull_request_template.md

```diff
@@ -0,0 +1,98 @@
+# SC-###
+
+<!--
+    If applicable, insert the Shortcut story number in the markdown header above.
+    The hyperlink will be filled in by GitHub magic ([autolink references](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources))
+--->
+
+## Proposed changes
+
+<!-- description and/or list of proposed changes -->
+
+---
+
+<!--
+    Please add/remove/edit any of the template below to fit the needs
+    of this specific PR
+--->
+
+## Reviewer notes
+
+<!--
+    Is there anything you would like reviewers to give additional scrutiny?
+--->
+
+## Setup
+
+<!--
+    Add any steps or code to run in this section to help others run your code:
+
+    ```sh
+    echo "Code goes here"
+    ```
+--->
+
+---
+
+## Code review steps
+
+### As the original developer, I have
+
+- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
+- [ ] Created new stories in Storybook if applicable
+  - [ ] Checked that all Storybook accessibility checks are passing
+- [ ] Created/modified automated unit tests in Jest
+  - [ ] Including jest-axe checks when UI changes
+- [ ] Created/modified automated E2E tests in Cypress
+  - [ ] Including cypress-axe checks when UI changes
+- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
+  - [ ] Checked responsiveness in mobile, tablet, and desktop
+  - [ ] Checked keyboard navigability
+  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
+  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
+  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)
+- [ ] Requested a design review for user-facing changes
+- For any new migrations/schema changes:
+  - [ ] Followed guidelines for zero-downtime deploys
+
+### As code reviewer(s), I have
```

Comment:
Given this is a PR template, how does this work with multiple reviewers?

Reply:
I don't know that the checkboxes are meant to literally be filled out, but rather serve as a list for a reviewer to go down when they're reviewing